homeassistant/components/wemo/sensor.py

```diff
@@ -78,6 +70,13 @@ def unique_id(self) -> str:
         """Return the id of this entity."""
         return f"{self.wemo.serialnumber}_{self._name_suffix}"
```

Comment:
I wonder if you intended this to be `insight_params_key` instead?

Reply:
Good idea.